app/views/record/_extended_info.html.erb

```diff
@@ -4,91 +4,83 @@
   <% search_prefix = ENV['EDS_PROFILE_URI'] %>
 <% end %>
 
-<h3 class="subtitle3">More information</h3>
-
-<ul>
+<h3 class="section-title">More information</h3>
+<ul class="list-moreinfo">
   <% if @record.eds_publication_type.present? %>
     <li>
-      Document type: <%= @record.eds_publication_type %>
-    </li>
+      <span class="label">Document type:</span> <%= @record.eds_publication_type %>
+    </li> 
   <% end %>
 
   <% if @record.eds_source_title.present? %>
     <li>
-      Source: <%= @record.eds_source_title %>
-    </li>
+      <span class="label">Source:</span> <%= @record.eds_source_title %>
+    </li> 
   <% end %>
 
   <% if @record.eds_authors.present? %>
     <li>
-      <ul>
-        <% @record.eds_authors.each do |author| %>
-          <li>
-            <%= author %>
-          </li>
-        <% end %>
-      </ul>
-    </li>
+      <span class="label">Authors:</span> 
+      <% @record.eds_authors.each do |author| %>
+        <span class="record-author"><%= author %></span>
+      <% end %>
+    </li> 
   <% end %>
 
   <% if clean_affiliations.present? %>
     <li>
-      <ul>
-        <% clean_affiliations.each do |aff| %>
-          <li>
-            <%= aff %>
-          </li>
-        <% end %>
-      </ul>
-    </li>
+      <span class="label">Affiliations:</span> 
+      <% clean_affiliations.each do |aff| %>
+        <%= aff %>; 
+      <% end %>
+    </li> 
   <% end %>
 
   <% if @record.eds_publication_info.present? %>
     <li>
-      Publication info: <%= safe_output(@record.eds_publication_info) %>
-    </li>
+      <span class="label">Publication info:</span> <%= safe_output(@record.eds_publication_info) %>
+    </li> 
   <% end %>
 
   <% if @record.eds_isbn_print.present? %>
     <li>
-      ISBN: <%= @record.eds_isbn_print %>
-    </li>
+      <span class="label">ISBN:</span> <%= @record.eds_isbn_print %>
```

Comment:
FYI in another branch I list all the ISBNs (and ISSNs). How about we let this branch merge *first*; then I'll rebase the other branch onto this one and you can tidy up the aesthetics as needed.

Reply:
Deal.